src/pages/Home.js

```diff
@@ -18,33 +20,37 @@ const StyledContainer = styled(ContentWrapper)`
   }
 `;
 /* use this to feed with mockData */
-const data = { products: mockData.items.slice(0, 6) };
+// const data = { products: mockData.items.slice(0, 6) };
 
 const Home = () => {
-  // const [data, setData] = useState({ products: [] });
-  // const [isFetching, setIsFetching] = useState(false);
-  // useEffect(() => {
-  //   const fetchData = async () => {
-  //     try {
-  // setIsFetching(true);
-
-  //       setData({ products: data.products});
-  //       const response = await axios.get(API_URL);
-  // setIsFetching(false);
-
-  //       setData({ products: response.data.items });
-  //     } catch (e) {
-  // setIsFetching(false);
-
-  //       setData({ products: data.products });
-  //     }
-  //   };
-  //   fetchData();
-  // }, []);
+  const [data, setData] = useState({ products: [] });
```

Comment:
You can change this state name to be "products" and store it as an array since you only have one property on data right now.

Reply:
noted!